examples/chat_app/README.md

```diff
@@ -0,0 +1,63 @@
+# LLM Chat UI <!-- omit in toc -->
+This is a chat demo using the various versions of the LLMs
+
+**Contents**:
+- [set up](#set-up)
+- [Launch the app](#launch-the-app)
+
+# set up
+
+1. Install **onnxruntime-genai-directml** 
```

Comment:
Could you add a sentence like that: "The app supports all backend that onnxruntime-genai supports. DirectML is used as an example to showcase how to use it"